### PR TITLE
add path run_time plan & unit case

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

**Review by CPWstatic — DISMISSED** [review]
LGTM.

**Review by CPWstatic — APPROVED** [review]

File: tests/query/v1/test_find_path.py [hunk 1]
```diff
@@ -379,3 +379,158 @@ def test_multi_source_no_path(self):
...
+
+    def test_run_time_input(self):
+        stmt = '''YIELD "Yao Ming" AS src, "Tony Parker" AS dst
+                | FIND SHORTEST PATH FROM $-.src TO $-.dst OVER like, serve UPTO 5 STEPS'''
+        resp = self.execute_query(stmt)
+        self.check_resp_succeeded(resp)
+        expected_data = {
+            "column_names": ["_path"],
+            "rows": [
+                [b"Yao Ming", (b"like", 0, b"Shaquile O'Neal"), (b"like", 0, b"Tim Duncan"), (b"like", 0, b"Tony Parker")],
```
Comment (yixinglu) [inline tests/query/v1/test_find_path.py:391]: Why not handle the binary string in check_path_xxx function for building the string's expected result easily.

**Review by yixinglu — COMMENTED** [review]
Others LGTM. Maybe we need some documents to explain the steps of plan generation for FIND PATH concisely.

File: tests/query/v1/test_find_path.py (hunk 1 above)
Comment (nevermore3) [reply tests/query/v1/test_find_path.py]: done

**Comment by nevermore3** [comment]
> Others LGTM. Maybe we need some documents to explain the steps of plan generation for FIND PATH concisely.

writing it

**Review by yixinglu — APPROVED** [review]

**Review by dutor — APPROVED** [review]
LGTM.

**Pull request merged**